README.md

```diff
@@ -113,6 +113,24 @@ Elixir's 7 basic types:
 * `lists`
 * `tuples`
 
+### Truthiness in Elixir
+
+All values / types in Elixir apart from `nil` and `false` are truthy:
+
+```ex
+if 0 do
+  "This is happening" 
+end # "This is happening"
+
+if "False" do
+  "This is still happening"
+end # "This is still happening"
+
+if false do
+  "Not happening"
+end # nil
+```
```

Comment:
I'm not sure if you typically want to have just a lone `if statement, do: something`, it would normally be surrounded by other statements which you could related to it with `if`s or other comparison functions.

In regards to having these 3 `if `statements next to each other, it sort of looks similar to having many of the same functions next to each other which return simple values. The best practices for these are described [here](https://github.com/christopheradams/elixir_style_guide#single-line-defs)